memory/empty_memory.go

```diff
@@ -0,0 +1,32 @@
+package memory
+
+import (
+	"github.com/tmc/langchaingo/schema"
+)
+
+// Empty memory is a class that implement the memory interface, but does nothing.
+// The class is used as default in multiple chains.
```

Comment:
I see "SimpleMemory" in the python codebase but not an empty implementation -- should this be Simple vs Empty?

Reply:
Yes, empty was just a name I came up with. Didnt know somthing like it existed in the python or ts version. Empty output parser should also be renamed.